app/views/users/profile.html.erb

```diff
@@ -296,6 +298,48 @@
       $("#social-links").toggle();
   });
 
+  $(document).ready(function() {
+    function splitText(text) {
+        var textBreak = textLimit;
+        var first = text.substring(0, textBreak);
+        var second = text.substring(textBreak);
+        var aux = second.substring(0, second.indexOf(" "));
+        var spaceIndex = second.indexOf(" ");
+        second = " " + second.substring(spaceIndex + 1);
+        first = first.substring(0, textBreak) + aux;
+        var bothTextes = [first, second];
+        return bothTextes;
+    }
+
+
+
+    var textLimit = 150;
+    var text = $("#user-bio").html();
+
+
+    if (text.length > textLimit) {
+        var textArray = splitText(text);
+        $("#user-bio").text(textArray[0]);
+        $("#user-bio").append("<span onclick=\"expandText()\" class='show-more'>...<span class=\"red\">show more</span></span>");
```

Comment:
It can be written in a better way using ES6 syntax of by using `single quote` and `double quote` interchangibly.
For example:
`single quote double quote` method:
```js
'<span onclick="expandText()" class="show-more">...<span class="red">show more</span></span>'
```
`ES6 syntax`
```js
`<span style="display: none" class="rest-of-description"> ${textArray[1]} </span>`
```